tests/core/tests/test_resources.py

```diff
@@ -1354,6 +1355,39 @@ def test_rollback_on_validation_errors_true(self):
         # Ensure the rollback has worked properly, no instances were created.
         self.assertFalse(Author.objects.exists())
 
+    def test_rollback_auxiliary_on_commit_transactions_when_dry_run(self):
```

Comment:
We probably need an additional test to ensure that rollback is not called when `using_transactions` is False